sections/apps.liquid

```diff
@@ -1,4 +1,4 @@
-<div class="{% if section.settings.include_margins %}page-width{% endif %}">
+<div class="{% if section.settings.include_margins %}page-width{% endif %}{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
```

Comment:
I guess some apps could be about adding image banners or images in general that could use a fade in instead of slide in animation. But I don't think we need to add a setting to the section at this point. We can see if it over comes up through our support channel as a request 👍